package.json

```diff
@@ -29,7 +29,9 @@
     "build-umd": "rimraf umd && cross-env NODE_ENV=production webpack --entry ./src/index.ts -o umd",
     "build-types": "rimraf types && npm run tsc",
     "demo": "webpack serve --config demo/webpack.config.js --port 3000 --host 127.0.0.1 --progress --profile --static demo/",
-    "test": "cross-env NODE_ENV=test karma start test/karma.conf.js",
+    "test": "npm run test-karma",
```

Comment:
How about running both tests in CI?

Reply:
Oops, that was the intent. What I get for making a PR tired 😄